Emby.Server.Implementations/Services/HttpContextExtension.cs

```diff
@@ -0,0 +1,33 @@
+using MediaBrowser.Model.Services;
+using Microsoft.AspNetCore.Http;
+
+namespace Emby.Server.Implementations.Services
+{
+    /// <summary>
+    /// Extention to enable the service stack request to be stored in the HttpRequest object.
```

Comment:
I think it would be better to describe this class as a generic place to store any extension method for `HttpContext`, something like this:

```suggestion
    /// Static class containing extension methods for <see cref="HttpContext"/>.
```